src/Microsoft.ML.Core/ComponentModel/ComponentCatalog.cs

```diff
@@ -1110,12 +1110,7 @@ private void LoadExtensions(Assembly assembly, bool throwOnError)
         /// </remarks>
         private static bool CanContainExtensions(Assembly assembly)
         {
-            // We used to have two separate strong name keys for product and test binaries and Arcade
-            // does not support that anymore (and that did not accept our PR for adding the additional
-            // signed key and using two strong name keys without the PR). If they ever decide to support
-            // this scenario we can remove the regex and go back to checking just the public key and startswith.
             if (assembly.FullName.StartsWith("Microsoft.ML.", StringComparison.Ordinal)
-               && !Regex.IsMatch(assembly.FullName, @"Microsoft\.ML.*\.Tests.*")
                && HasMLNetPublicKey(assembly))
```

Comment:
(nit) your previous change had some whitespace issues going from 4 spaces to 3 on this line, and on the `return false;` line below. Can you fix those while you are in here?

Reply:
Not sure how that happened but its fixed now.